parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -142,6 +142,18 @@ configureCustomCaCertificate() {
     systemctl restart containerd
 }
 
+configureContainerdUlimits() {
+  CONTAINERD_ULIMIT_DROP_IN_FILE_PATH="/etc/systemd/system/containerd.service.d/set_ulimits.conf"
+  touch "${CONTAINERD_ULIMIT_DROP_IN_FILE_PATH}"
+  chmod 0600 "${CONTAINERD_ULIMIT_DROP_IN_FILE_PATH}"
+  tee "${CONTAINERD_ULIMIT_DROP_IN_FILE_PATH}" > /dev/null <<EOF
+$(echo "$CONTAINERD_ULIMITS" | tr ' ' '\n')
```

Comment:
you're already formatting it in go, why do extra formatting here?

or why not template it entirely? see containerd config for example if special chars is hard (use b64)

Reply:
I formatted it here because new lines were not getting transported over as expected, will try with b64